CHANGELOG.md

```diff
@@ -1,29 +1,67 @@
 # Change Log
+
 All notable changes to this project will be documented in this file.
-See the [change log guidelines](http://keepachangelog.com/) for information on how to structure the file.
+See the [change log guidelines](https://github.com/halestudio/hale/compare/5.0.1...5.1.0) for information on how to structure the file.
+
+## [Unreleased](https://github.com/halestudio/hale/compare/5.0.1...5.1.0)
+
+## [5.1.0]
 
-## [Unreleased]
+### Added
+
+- Support for reading multiple sheets from an Excel file simultaneously
+- Enable exporting of multiple feature types to Excel
+- Allow loading Excel with blank rows and sheets for schema and source data
+- Add predefined and custom formatters for Date cells in Excel import
+- Enable exporting feature types with no data in Excel
+- Align Excel with CSV export wizard
+- Prefer properties with short paths as GeoJSON geometry property
+- Add wizard page to select the read mode for GeoJSON/JSON
+- Added support for importing data from JSON files, enhancing the versatility of data import options.
```

Comment:
Added support for importing data from JSON files --> Support for importing data from JSON files

Reply:
Allow loading Excel with blank rows and sheets for schema and source data --> Allow loading Excel which contains blank rows and sheets for schema and source data. Instances are not added for blank entries.